fmriprep/workflows/epi.py

```diff
@@ -148,7 +149,11 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
             ('outputnode.acompcor_report', 'inputnode.acompcor_report'),
             ('outputnode.tcompcor_report', 'inputnode.tcompcor_report')]),
     ])
-
+    if use_aroma:
+        workflow.connect([
+            (discover_wf, func_reports_wf, [
+                ('outputnode.ica_aroma_report', 'inputnode.ica_aroma_report')]),
+        ])
```

Comment:
This doesn't need to be in an `if` block. You can just add it to the `(discover_wf, func_reports_wf, ...)` tuple above.